src/scrabble.js

```diff
@@ -1,5 +1,51 @@
-function scrabble() {
-  // write code here
+const scrabble = (myWord) => {
```

Comment:
I really like that you went with storing all the letter values as key-value pairs in an object - it's a lot more concise than my 7 arrays, and I can see how that also allows for the rest of your code to also be quite a bit more concise than what I went for.